_posts/2019-05-20-mockiato-announcement.md

```diff
@@ -158,13 +158,14 @@ mod tests {
 
 ## Limitations
 ### `Sync` / `Send`
-Mocks are neither sync, nor send.
+Mocks are neither sync, nor send. The tracking issue for this limitation is [#106](https://github.com/myelin-ai/mockiato/issues/106).
```

Comment:
Why not simply link them like `[Tracking issue](https://github.com/myelin-ai/mockiato/issues/106)` ?

Reply:
17bcf2e